docs/csharp/language-reference/operators/snippets/shared/u2ngxcbv.ld0~

```diff
@@ -0,0 +1,141 @@
+﻿using System;
```

Comment:
Is this file unintentionally checked in?

Reply:
Good catch. No. It shouldn't be.